dockstore-client/src/main/java/io/dockstore/common/VersionAwarePluginManager.java

```diff
@@ -80,7 +82,7 @@ void cleanupOldVersions() {
         Set<File> fileSet = pluginVersionMap.values().stream().map(Pair::getRight).collect(Collectors.toSet());
         for (File directory : directories) {
             if (!fileSet.contains(directory)) {
-                LOG.debug("Deleting old version of plugin in {}", directory.getAbsoluteFile());
+                LOG.debug("Deleting old version of " + PLUGIN + " in {}", directory.getAbsoluteFile());
```

Comment:
Can take advantage of log message formatting by using `{}`

Reply:
I have now done this, thanks!